content/source/docs/index.html.markdown

```diff
@@ -10,70 +10,35 @@ show_notification: false
 
 # Terraform Documentation
 
-Welcome to the Terraform documentation!
-
-We've divided the Terraform docs into several sections to make it easier to find things. Navigate using this index page, or jump from section to section in the "Other Docs" section of the navigation sidebar.
-
 <div class="container-fluid"><div class="row">
-
 <div class="col-md-6 col-sm-12">
 
-### [Introduction to Terraform ➜](/intro/index.html)
-
--> Visitors who are curious about Terraform can start here.
-
-A broad overview of what Terraform is and why people use it.
-
-### [Terraform Glossary ➜](/docs/glossary.html)
-
-Definitions (and helpful links) for technical terms used throughout Terraform's documentation, help text, and UI. Visit the glossary whenever you get lost.
-
-### [Configuration Language ➜](/docs/language/index.html)
-
--> Intermediate and advanced users spend most of their time here.
-
-Documentation for Terraform's configuration language.
+### Get Started
 
-The Terraform configuration language is Terraform's primary user interface. This section is relevant to all users of Terraform, including Terraform Cloud and Terraform Enterprise users.
+- Learn how Terraform [solves infrastructure challenges](/intro/index.html) and how it [compares to other tools and services](/intro/vs/index.html).
+- [Follow the tutorials on HashiCorp Learn](https://learn.hashicorp.com/collections/terraform/aws-get-started?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) to install Terraform, learn the Terraform workflow, and explore use cases.
 
-### [Terraform CLI ➜](/docs/cli/index.html)
 
-Documentation for Terraform's command-line workflows, including docs for [the `terraform` binary and its subcommands](/docs/cli/commands/index.html).
+### Manage Infrastructure
 
-### [Learn Terraform ➜](https://learn.hashicorp.com/terraform?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) (external site)
+- Use the [Terraform Configuration Language](/docs/language/index.html) to describe infrastructure.
+- Use the [Terraform CLI](/docs/cli/index.html) with the Terraform binary for command-line workflows.
```

Comment:
```suggestion
- Use the [Terraform Command Line Interface (CLI)](/docs/cli/index.html) to manage configuration, plugins, infrastructure, and state.
```

Reply:
I think there's a world where we separate docs for the CLI and the binary itself, but we aren't there yet, and you can use many of the CLI commands with Terraform Cloud too :)